src/main/java/com/egen/repository/OrderRepositoryImpl.java

```diff
@@ -0,0 +1,210 @@
+        query.setParameter("paramZip",zip);
+        List<Order> list=query.setMaxResults(10).getResultList();
+        return list;
+    }
+
+    @Override
+    public List<Order> getAllOrdersWithInInterval(Timestamp startTime, Timestamp endTime){
+        Query query = em.createQuery("SELECT ord FROM Order ord  WHERE ord.createdDate BETWEEN :ordStartTime AND :ordEndTime");
+        query.setParameter("ordStartTime",startTime);
+        query.setParameter("ordEndTime",endTime);
+        List<Order> list=query.getResultList();
+        return list;
+    }
+    @Override
+    public List<Order> findAll(Sort sort) {
+        return null;
+    }
+
+    @Override
+    public Page<Order> findAll(Pageable pageable) {
+        return null;
+    }
+
+    @Override
+    public List<Order> findAllById(Iterable<Long> iterable) {
+        return null;
+    }
+
+    @Override
+    public long count() {
+        return 0;
+    }
+
+    @Override
+    public void deleteById(Long aLong) {
+
+    }
+
+    @Override
+    public <S extends Order> S save(S s) {
+        return null;
+    }
+
+    @Override
+    public <S extends Order> List<S> saveAll(Iterable<S> iterable) {
+        return null;
+    }
+
+    @Override
+    public Optional<Order> findById(Long aLong) {
+        return Optional.empty();
+    }
+
+    @Override
+    public boolean existsById(Long aLong) {
+        return false;
+    }
+
+    @Override
+    public void flush() {
+
+    }
+
+    @Override
+    public <S extends Order> S saveAndFlush(S s) {
+        return null;
+    }
+
+    @Override
+    public <S extends Order> List<S> saveAllAndFlush(Iterable<S> iterable) {
+        return null;
+    }
+
+    @Override
+    public void deleteAllInBatch(Iterable<Order> iterable) {
+
+    }
+
+    @Override
+    public void deleteAllByIdInBatch(Iterable<Long> iterable) {
+
+    }
+
+    @Override
+    public void deleteAllInBatch() {
+
+    }
+
+    @Override
+    public Order getOne(Long aLong) {
+        return null;
+    }
+
+    @Override
+    public <S extends Order> Optional<S> findOne(Example<S> example) {
+        return Optional.empty();
+    }
+
+    @Override
+    public <S extends Order> List<S> findAll(Example<S> example) {
+        return null;
+    }
+
+    @Override
+    public <S extends Order> List<S> findAll(Example<S> example, Sort sort) {
+        return null;
+    }
+
+    @Override
+    public <S extends Order> Page<S> findAll(Example<S> example, Pageable pageable) {
+        return null;
+    }
+
+    @Override
+    public <S extends Order> long count(Example<S> example) {
+        return 0;
+    }
+
+    @Override
+    public <S extends Order> boolean exists(Example<S> example) {
+        return false;
+    }
+
+    @Override
+    public void deleteAllById(Iterable<? extends Long> iterable) {
+
+    }
+
+    @Override
+    public void deleteAll(Iterable<? extends Order> iterable) {
+
+    }
+
+    @Override
+    public void deleteAll() {
+
+    }
+
+
```

Comment:
Good job here. I like that you have all these additional methods. Would be great if you implemented them as well.